docs/en/reference/schema-representation.rst

```diff
@@ -47,12 +50,13 @@ foreign key, sequence and index changes.
 .. code-block:: php
 
     <?php
+    /*...*/
     $schemaManager = $connection->createSchemaManager();
     $comparator = $schemaManager->createComparator();
-    $schemaDiff = $comparator->compare($fromSchema, $toSchema);
+    $schemaDiff = $comparator->compareSchemas($fromSchema, $toSchema);
 
-    $queries = $schemaDiff->toSql($myPlatform); // queries to get from one to another schema.
-    $saveQueries = $schemaDiff->toSaveSql($myPlatform);
+    $createdSchemas = $schemaDiff->getCreatedSchemas();
```

Comment:
This doesn't look like a reasonable replacement. The previous example returned the queries to migrate to the new schema, while the new one returns schema names. See

https://github.com/doctrine/dbal/blob/750d846074fe920b65dcfca3c9bdcea45fa78a97/UPGRADE.md#L162-L165